.github/workflows/deploy.yml

```diff
@@ -37,6 +37,29 @@ jobs:
       - run: yarn install
       - run: yarn build
 
+      - name: Install Capacitor Dependencies
+        run: npm install @capacitor/core @capacitor/cli @capacitor/android
```

Comment:
@coderabbitai - would this not already resolved by yarn install / yarn build as they are included in package.json already?